src/syncer.py

```diff
@@ -133,6 +133,9 @@ def _sync(self):
                 continue
 
             mail = mailingList["mail"]
+            if mail.startswith("p_"):
+                mail = mail[2:]
```

Comment:
I don't like this. As @PLanB2008 mentioned previously, there can be conflicts by blindly doing this.
Also, it will change currently existing lists! This is unacceptable because it will break the setup for everyone who is using the lists with p_ prefix.

I could imagine adding an option to automatically create aliases without the p_ prefix, as this would not break existing setups and would have to be enabled manually.